eZ/Publish/Core/MVC/Symfony/EventListener/ContentViewSubscriber.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\EventListener;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\ExpressionLanguage\ExpressionLanguage;
+use eZ\Publish\Core\MVC\Symfony\ExpressionLanguage\TwigVariableProviderExtension;
+use eZ\Publish\Core\MVC\Symfony\Event\PreContentViewEvent;
+use eZ\Publish\Core\MVC\Symfony\MVCEvents;
+use eZ\Publish\Core\MVC\Symfony\View\VariableProviderRegistry;
+use eZ\Publish\Core\MVC\Symfony\View\View;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+final class ContentViewSubscriber implements EventSubscriberInterface
+{
+    private const EXPRESSION_INDICATOR = '@=';
+
+    public const PARAMETERS_KEY = 'params';
```

Comment:
I don't like `params` name in this context, why not keeping it consistent with SA parser and call `twig_variables`? `params` feels like something related to controller or matching action.
I get that this comes from the concept of a _View_ which has _parameters_, so internally this is fine, but for an integrator writing config it should be something more meaningful.
... or maybe the other way around, instead `twig_variables` in SA parser, let's call it `view_parameters`?